### DISCUSSION TITLE
Corner Ornament component update

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#4027] [General] [answered]
Hi,

On the roadmap it had said the Corner Ornament would be ready 8/6. Wondering what the current status is?

**Comment by serifluous** [comment, marked as answer]
Hey @pamelahu ! We did actually end up deciding to ship Corner Ornament as a component instead of a pattern. The component will be released as part of this PR: #3992 within the next week.

If you're interested in a sneak peek, you can check out the docs here right now: https://paste.twilio.design/components/corner-ornament

**Reply by serifluous** [reply]
Hey @pamelahu ! Corner Ornament is now available in Paste.
